config/_default/menus/menus.en.toml

```diff
@@ -1,6 +1,6 @@
 [[main]]
   name = "Docs"
-  url = "/docs/overview/about"
+  url = "/docs/"

```

Comment:
is this relink intended? this might break urls right?

Reply:
It is not really a relink it is just the menu item for "Docs" in the header of the website. Document links in Hugo are by default constructed from the location it has under content and then its title (spaces and underscores converted to "-"). There's also the ability to use the "url" attribute in the frontmatter to explicitly define a URL. This is for example how I did it for the community page.

Long story short: this will not break any link of existing documents.